packages/firestore/src/platform/browser/webchannel_connection.ts

```diff
@@ -208,7 +209,9 @@ export class WebChannelConnection extends RestConnection {
     }
 
     if (this.useFetchStreams) {
-      request.useFetchStreams = true;
+      // When b/307942499 is fixed, revert to the following line:
```

Comment:
The [convention](https://cs.opensource.google/search?q=%22%2F%2F%20TODO\(b%22&sq=&ss=firebase-sdk%2Ffirebase-js-sdk) for formatting such comments is:
```js
// TODO(b/307942499): switch to `useFetchStreams` once WebChannel is fixed.
```

Reply:
Thanks.